### ISSUE TITLE
Current Boosts: Should overload expiring trigger "boosted stats below" notification?

### ISSUE THREAD

**Issue by hctim852** [rsbmatt/OSBuddy#17]
Normally when a boosted stats drop below whatever you set as your threshold, whether you brew down or it ticks down normally, it triggers the notification. (E.g. You are 80 attack and set your notification threshold at 85) Overloads expiring don't trigger this notification.

v3.0.10

**Comment by rsbfox** [comment]
Hey hctim852,

Thanks for using the new issue tracker. Please include client version string from top of client in post. 
Please also prefix issue title with "Plugin Name: " in the future.

Thanks!

Also what is your discord tag?

**Comment by hctim852** [comment]
Will make sure to add those for future posts. Mitch#2001.

**Comment by rsbfox** [comment]
Okay. Confirmed overloads do not trigger those alerts. But I think this could be marked as intended instead.

I think a separate alert under current boosts or nightmare zone plugin for overload expiring would be better. Else you need to change values every time you switch between overloads and supers.

Discussion: What would be best solution?

**Comment by hctim852** [comment]
Agreed that it should maybe be added to the nightmare zone plugin and then tied into the current boosts status indications. Should be okay to split them up, as the current boosts indicators and current active prayer indicators are split into separate plugins. 

Further, if you have an overload active, should the current boost displayed just be an overload icon with timer, or should it show all boosted skills (att, str, def, range, mage) as separate icons? I'd prefer just having a single overload icon with timer, else the boost/timers overlay would be really cluttered with icons.

**Comment by rsbfox** [comment]
Good suggestions. Labeling this enhancement.
